src/Microsoft.ML.DnnImageFeaturizer.AlexNet/AlexNet.cs

```diff
@@ -0,0 +1,54 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.ML.Runtime;
+using Microsoft.ML.Runtime.Data;
+using System.IO;
+using System.Reflection;
+
+namespace Microsoft.ML.Transforms
+{
+    /// <summary>
+    /// This is an extension method to be used with the <see cref="DnnImageFeaturizerEstimator"/> in order to use a pretrained AlexNet model.
+    /// The NuGet containing this extension is also guaranteed to include the binary model file.
+    /// </summary>
+    public static class AlexNetExtension
```

Comment:
>AlexNetExtension [](start = 24, length = 16)

(nit) it is nice to keep the class name and the containing file name in sync.

`AlexNetExtension` vs `AlexNet.cs`